cypress/tests/mocked/parallel-1/updateContributionAmount.cy.ts

```diff
@@ -98,6 +101,10 @@ describe('Update contribution amount', () => {
 
 		cy.findByText('Change amount').click();
 
+		cy.contains(
+			/£\d{2,3} per month is the minimum payment to receive this subscription./,
```

Comment:
Are we fuzzy matching the amount here to account for future config changes? (I.e. so this test doesn't have to be updated each time the config changes?)

Reply:
yeah I fuzzy matched here to reduce the number of places that you would have to update if/when we do another price rise ... there are still places where you have to do it manually though, so this doesn't solve that 'problem' everywhere, only here.